.travis.yml

```diff
@@ -8,15 +9,24 @@ env:
 matrix:
   include:
     - language: python
-      name: Check Boilerplate 
+      name: Code lint - python
```

Comment:
we don't really do minikube code linting, all we really do is checking for license boilerplates. we could rename it to "boilerplate lint".